pyproject.toml

```
@@ -62,7 +62,7 @@ pyinstaller = "^5.6,<5.7.0" # NOTE: 5.7.0 or higher will fail to build the bootl
 pre-commit = "^2.16.0"
 atomicwrites = "^1.4.0"
 colorama = "^0.4.4"
-poetry = "^1.3.1"
+poetry = "^1.5.1,>=1.5"
```

`,>=1.5`の意図はなんでしょうか 👀 
あってもなくても良いなら周りに合わせた方が好ましそうです！
```suggestion
poetry = "^1.5.1"
```

（PRの範囲に関係ないのですが、そもそも範囲指定じゃなく単一バージョンに固定したほうが確実かも？とかちょっと思いました！）

---

開発者がPoetryを使った操作をするときに、Poetry 1.4以下を使わないようにするためですが、あんまり意味はないかもです。

バージョン`1.5.x`を使ってほしい場合には、`~1.5.1`のように`1.5 <= version < 1.6`を指定できます。
`^1.5.1`の場合、`1.0 <= version < 2.0`になります。
以前の指定では、`1.0 <= version < 2.0`だったので、`1.5 <= version < 2.0`に範囲を狭めました。

ちなみに、開発者がシステムにPoetryをインストールしている場合、`pyproject.toml`や`poetry.lock`に含まれるPoetryバージョンには制限を受けなさそうです。

これらが管理するのは、`poetry install`や`pip install -r requirements-dev.txt`で（venvに）インストールされるPoetryのバージョンで、開発者がそのPoetryを操作に使わなかった場合には影響しないと思います。

`pyproject.toml`を見た開発者に、このバージョンを使ってほしいという意図を伝えることはできるかもですが、操作時にエラーを出すような形で固定できるわけではないと思います。

`poetry install`して（システムの）Poetryが作成した仮想環境に入れば、venvのPoetry（`poetry.lock`で管理されたバージョンのPoetry）が使われるようになるようなので、Poetry 1.5以上を使われやすくする効果はあると思います。